README.md

```diff
@@ -8,21 +8,84 @@
 
 AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and implements cluster provisioning and lifecycle operations for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
 
-More info, including a thorough walkthrough is [here](docs/aksengine.md).
+## Install AKS-Engine
 
-Please see the [FAQ](/docs/faq.md) for answers about AKS-Engine and its progenitor ACS-Engine.
+To install via homebrew, run `brew tap azure/aks-engine && brew install azure/aks-engine/aks-engine` or download the binary via the [github releases page](https://github.com/Azure/aks-engine/releases)
```

Comment:
we also have [dockerhub](https://hub.docker.com/r/microsoft/acs-engine/) and [gofish](https://github.com/fishworks/fish-food/blob/master/Food/acs-engine.lua) as less-than-ideally maintained current options for installing/using acs-engine, if we want to keep those around for aks-engine we should make them automated and more officially supported.